Formula/a/austin.rb

```diff
@@ -31,13 +31,12 @@ def install
   end
 
   test do
-    if OS.mac?
-      assert_match "Insufficient permissions. Austin requires the use of sudo",
-        shell_output("#{bin}/austin --gc 2>&1", 37)
+    expected = if OS.mac?
+      "Insufficient permissions. Austin requires the use of sudo"
     else
-      assert_match "need either a command to run or a PID to attach to",
-        shell_output("#{bin}/austin --gc 2>&1", 255)
+      "need either a command to run or a PID to attach to"
     end
```

Comment:
I dont think it would work? Somehow there is no output for `austin --gc` for 4.0.0 release.